java6/src/main/java/io/papermc/paperclip/Paperclip.java

```diff
@@ -13,7 +13,8 @@
 
     public static void main(final String[] args) {
         System.err.println("Minecraft 1.17 requires running the server with Java 16 or above. " +
-            "Download Java 16 (or above) from https://adoptopenjdk.net/");
+            "Download Java 16 (or above) from https://adoptium.net/. Fore more information, see " +
```

Comment:
> Fore

and no, my suggestion didn't had that typo :D

Reply:
Your grammar skills here are surely convincing

Reply:
Please don't harass your contributors, otherwise I need to report you to the moderation team. Thanks in advance.